### PR TITLE
Fix registry images

### PR DESCRIPTION
**Pull request by cynepco3hahue**
The latest changes under the base registry images broke our build because
the base image does not have any binaries except ones that related to
the registry under the `/bin` directories. Also, the default user
of the registry base image changed from the root to 1001, that will prevent
to save `bundles.db` database under the root directory.

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: openshift-ci/Dockerfile.registry.build [hunk 1]
```diff
@@ -11,8 +15,13 @@ ENV REG_URL=registry.svc.ci.openshift.org
 
 # replaces performance-addon-operator image with the one built by openshift ci
 RUN find /registry/performance-addon-operator-catalog/ -type f -exec sed -i "s|REPLACE_IMAGE|${REG_URL}/${OPENSHIFT_BUILD_NAMESPACE}/stable:performance-addon-operator|g" {} \; || :
+
 # Initialize the database
+USER root
```
Comment (fromanirh) [inline openshift-ci/Dockerfile.registry.build:20]: why is this needed now?

File: openshift-ci/Dockerfile.registry.upstream.dev [hunk 2]
```diff
@@ -9,7 +13,11 @@ COPY ${OLM_SOURCE} /registry/performance-addon-operator-catalog
 RUN find /registry/performance-addon-operator-catalog/ -type f -exec sed -i "s|REPLACE_IMAGE|${FULL_OPERATOR_IMAGE}|g" {} \; || :
```
Comment (fromanirh) [inline openshift-ci/Dockerfile.registry.upstream.dev:13]: another option is getting rid of this entirely, possibly enhancing our `csv-replace-imageref` tool and/or packing in a different image

File: openshift-ci/Dockerfile.registry.build (hunk 1 above)
Comment (cynepco3hahue) [reply openshift-ci/Dockerfile.registry.build:20]: Because the base image now runs with USER 1001, but we trying to put the `bundle.db` under the `/` and the user 1001 not allowed to write under it.

File: openshift-ci/Dockerfile.registry.upstream.dev (hunk 2 above)
Comment (cynepco3hahue) [reply openshift-ci/Dockerfile.registry.upstream.dev:13]: The problem, not just a find or sed binaries, but the call to the `RUN` at all, because under the neat `RUN` calls to `/bin/sh -c`.

But to remove this line can make our Dockerfile cleaner, so I definitely not versus it, but it should be part of the different PR :)

File: openshift-ci/Dockerfile.registry.build (hunk 1 above)
Comment (fromanirh) [reply openshift-ci/Dockerfile.registry.build:20]: ok, so another change caused bu the base image changes :\

File: openshift-ci/Dockerfile.registry.upstream.dev (hunk 2 above)
Comment (fromanirh) [reply openshift-ci/Dockerfile.registry.upstream.dev:13]: ok, but if so every `RUN` instance will fail, not just this one, right?

Comment (cynepco3hahue) [reply openshift-ci/Dockerfile.registry.upstream.dev:13]: Yes, but to avoid it I just copy `sh` from /usr/bin to /bin

**Comment by cynepco3hahue** [comment]
Fixed by https://github.com/operator-framework/operator-registry/pull/437
